build.sbt

```diff
@@ -184,21 +184,24 @@ lazy val libSettings =
     )
   )
 
-lazy val gcSettings =
+lazy val gcSetting =
   if (!System.getenv.containsKey("SCALANATIVE_GC")) {
-    println("Using default gc")
-    Seq.empty
+    nativeGC := {
+      val gc = nativeGC.value
+      println(s"Using $gc gc")
+      gc
+    }
   } else {
     val gc = System.getenv.get("SCALANATIVE_GC")
     println(s"Using gc based on SCALANATIVE_GC=$gc")
-    Seq(nativeGC := gc)
+    nativeGC := gc
```

Comment:
Same here, it would be better to use `sLog`. Wouldn't it be clearer if written like this?

```scala
nativeGC := {
  val default = nativeGC.value
  val log = sLog.value
  val gc = Option(System.getenv.get("SCALANATIVE_GC")).getOrElse(default)
  log.info(s"Using $gc gc")
}
     
```

Reply:
That looks really nice, ready now. 👍